src/get_ranges.py

```diff
@@ -0,0 +1,25 @@
+"""Реализовать функцию get_ranges, которая получает на вхох непустой список
+неповторяющихся целых чисел, отсортированных по возрастанию, которая этот
+список сворачивает"""
+
+
+def get_ranges(lst):
+    out_str = ''
+    for el in range(len(lst)):
```

Comment:
maybe it's better to add additional check instead of try-except block?

Reply:
One of the principales of python is  ask forgiveness not permission. Because I think that using the try-except block is better.

Reply:
It can be in case of 2 conditions:
1. You know where and how would it be handled.
2. It's not attempt to hide error in algorithm.
In your case you wrote algorithm that would normally throw this exception. Exceptions are for handling exceptions, not a part of algorithm.